manifests/v/VoiceAttack/VoiceAttack/1.12/VoiceAttack.VoiceAttack.installer.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: VoiceAttack.VoiceAttack
+PackageVersion: "1.12"
+InstallerType: inno
+Installers:
+- Architecture: x86
+  InstallerUrl: https://www.voiceattack.com/filesend.aspx?id=voiceattackinstaller_112.exe
+  InstallerSha256: F2092CD0D32B2F64742BFE65A30DBF2658FEB293D7A596DF94E6B78689937648
+ManifestType: installer
+ManifestVersion: 1.5.0

```

Validated using the 1.6.0 JSON Schema that the rest of this manifest conforms:

```suggestion
ManifestVersion: 1.6.0
```